oemof/solph/components.py

```diff
@@ -63,8 +63,11 @@ class GenericStorage(network.Transformer):
         Couple storage level of first and last time step.
         (Total inflow and total outflow are balanced.)
     loss_rate : numeric (sequence or scalar)
-        The relative loss of the storage capacity from between two consecutive
+        The relative loss of the storage capacity between two consecutive
         timesteps.
+    fixed_losses : numeric (sequence or scalar)
+        Losses as share of nominal storage capacity between two consecutive
+        timesteps independent of storage capacity.
```

Comment:
Isn't it missleading to call it independent of storage capacity (naming discussion #538). I would say independent of storage_content.

Reply:
>  SoC is well know I think. "storage_content" is misleading to me as well, some could interpret this as water, oil, ...

For discussing this, please go to #538.

Reply:
I changed to 'state of charge' to avoid confusion.